site/lib/src/components/pages/changelog/breaking_changes_legend.dart

```diff
@@ -0,0 +1,104 @@
+// Copyright 2025 The Dart Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:jaspr/dom.dart';
+import 'package:jaspr/jaspr.dart';
+
+class BreakingChangesLegend extends StatelessComponent {
+  const BreakingChangesLegend({required this.onClose, super.key});
+
+  final VoidCallback onClose;
+
+  @override
+  Component build(BuildContext context) {
+    return div(classes: 'legend-overlay', [
+      div(classes: 'legend-dialog', [
+        div(classes: 'legend-header', [
+          const h3([.text('About Breaking Changes')]),
+          button(
+            classes: 'close-button',
+            onClick: onClose,
+            [
+              const span(classes: 'material-symbols-outlined', [
+                .text('close'),
+              ]),
+            ],
+          ),
```

Comment:
Can reuse our existing `Button` component here, which includes handling for an icon button.

```suggestion
          Button(
            icon: 'close',
            classes: ['close-button'],
            onClick: onClose,
          ),
```